.github/CODE_STYLE.md

```diff
@@ -118,6 +118,13 @@ export class TdsComponent {
 
 ```
 
+
+## Slots
+
+Slots are used throughout many of our components. Some of these are named slots, and in that case they should follow
+to following convention: The slot should always be named to represent it position/use case. For example this could be
```

Comment:
```suggestion
should follow the following convention: The slot should always be named to represent it position/use case. For example this could be
```